app/prisma/migrations/20260814200000_add_athena_action_idempotency/migration.sql

```
@@ -0,0 +1,59 @@
+  updated_at timestamptz not null default now(),
+  constraint athena_action_idempotency_completed_payload check (
+    (status = 'reserved' and action_json is null and result_json is null)
+    or
+    (status = 'completed' and action_json is not null and result_json is not null)
+  )
+);
+
+create unique index idx_athena_action_idempotency_scope
+  on athena_action_idempotency(org_id, tool_id, tool_version, idempotency_key);
+create index idx_athena_action_idempotency_actor_updated
+  on athena_action_idempotency(org_id, actor_user_id, updated_at);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)